pkg/apiserver/apiserver.go

```diff
@@ -141,9 +131,25 @@ func (c completedConfig) New() (*APIServer, error) {
 	groups := []*genericapiserver.APIGroupInfo{&networkingGroup, &systemGroup}
 	for _, apiGroupInfo := range groups {
 		if err := s.GenericAPIServer.InstallAPIGroup(apiGroupInfo); err != nil {
-			return nil, err
+			return err
 		}
 	}
+	return nil
+}
 
+func (c completedConfig) New() (*APIServer, error) {
+	genericServer, err := c.genericConfig.New("antrea-apiserver", genericapiserver.NewEmptyDelegate())
+	if err != nil {
+		return nil, err
+	}
+
+	s := &APIServer{
+		GenericAPIServer: genericServer,
+		caCertController: c.extraConfig.caCertController,
+	}
+
+	if err := installAPIGroup(s, c); err != nil {
+		return nil, err
+	}
```

Comment:
Why making this change?

Reply:
I think 1) it looks clearer and 2) it keeps the same code style like agent/apiserver.

Reply:
ok, but I think it's controversial to separate a short function into multiple ones when they are not reused, especially in a PR that is not related. We should focus on the purpose of the PR and leave unrelated style unification to separate PR to save reviewers' effort and avoid controversy.

Reply:
Make sense. I will pay more attention next time. I used to change some apiserver code in this PR, but such change is dropped with a comment. However, this code remains due to the reason I mentioned above..